qftasm.js

```diff
@@ -176,15 +176,15 @@ function step_code() {
         vals.push(val);
     }
     
+    PC[1] = PC[1] + 1;
     PC[0] = PC[1];
-    PC[1] = PC[0]+1;
-    if(RAMdisplay.indexOf(0) >= 0){ update_display(0); }
+    RAMwrite(0, PC[1], 0);

```

Comment:
`RAMwrite(0, PC[1], 0);` <- is this so that the program counter incrementing normally doesn't increment the write counter for address 0?